realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/AuthTests.java

```diff
@@ -661,15 +664,6 @@ public void run() {
     }
 
     @Test
-    public void retrieve_AdminUser() {
```

Comment:
this test is not needed? why?

Reply:
The `debug` auth provider is no longer supported, which this test was built around.